contrib/buildrpm.sh

```diff
@@ -57,6 +60,10 @@ if [ $opt_strict_ibverb_dep -eq 1 ]; then
     rpmmacros="${rpmmacros} --define='extra_deps libibverbs == ${libibverbs_ver}'"
 fi
 
+if [ $opt_dbgrpm -eq 1 ]; then
+    rpmmacros="--define 'devel 1' ${rpmmacros}"
+fi
```

Comment:
pls add the building of debug RPM to CI in here:
https://github.com/openucx/ucx/blob/898c2e56f94be35bc278a6f03a0b67c244233787/buildlib/tools/builds.sh#L87

Reply:
Done.
It seems that there's one bug here. Fix it in https://github.com/changchengx/ucx/commit/c97c436edf43ed5cfab2dd44b08b4c66e512fd1f